documentation/Admin/evm-key-rotation.md

```diff
@@ -0,0 +1,91 @@
+---
+id: evm-key-rotation
+sidebar_position: 1
+sidebar_label: EVM Key Rotation
+slug: /evm-key-rotation
+---
+# Master Key Rotation
+1. Generate the new master key to be rotated to.
+```
+axelard tx tss start-keygen --id {key ID} --from validator -y -b block --gas auto --gas-adjustment 1.2
+-> Wait until the new key is generated
+```
+The new key is successfully generated if the following query gives the key role `KEY_ROLE_MASTER_KEY`.
```

Comment:
is this already fixed in axelar-core? IIRC, the query would return the numeric values, not the constant names

Reply:
yup it's fixed.